services/venice-controller/src/main/java/com/linkedin/venice/controller/DeferredVersionSwapService.java

```diff
@@ -159,34 +144,6 @@ public void run() {
                   veniceParentHelixAdmin.getCurrentVersionsForMultiColos(cluster, storeName);
               Set<String> remainingRegions = getRegionsForVersionSwap(coloToVersions, targetRegions);
 
-              StoreResponse targetRegionStoreResponse =
```

Comment:
Do not delete this part yet. Lets add a config to enable this check, and disable it only after DVC clients are upgraded to include the new logic.